packages/contracts/contracts/L1/SecurityCouncil.sol

```diff
@@ -112,7 +102,9 @@ contract SecurityCouncil is MultiSigWallet, Semver {
             Colosseum.forceDeleteOutput.selector,
             _outputIndex
         );
-        uint256 transactionId = submitTransaction(COLOSSEUM, 0, message);
+        uint256 transactionId = submitTransaction(address(COLOSSEUM), 0, message);
+        // auto-confirmed by requester
+        confirmTransaction(transactionId);
```

Comment:
If `confirmTransaction` is included in `submitTransaction` as done in `MultiSigWallet.sol`, we don't need this.